src/device/pci/constants.rs

```diff
@@ -290,6 +290,23 @@ pub mod xhci {
             pub const CS: u64 = 0x2;
             pub const CA: u64 = 0x4;
         }
+
+        pub mod portsc {
+            /// Port power should always be enabled.
+            /// Software can only disable it.
+            const PP: u64 = 0x1 << 9;
```

Comment:
```suggestion
            const PP: u64 = 1 << 9;
```

nit: why the additional `0x`?

Reply:
Ah, yes. I changed that. Now it is consistent everywhere in this file.